### PR TITLE
Replace cl iby gdeploy calls

### PR DESCRIPTION
**Pull request by nnDarshan**

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

**Comment by r0h4n** [comment]
Please resolve merge conflicts and update if this PR is still required

**Comment by nnDarshan** [comment]
This is still needed...will resolve merge conflicts

**Comment by nnDarshan** [comment]
Have Updated the patch after rebase.

File: etc/create_sample_job.py [hunk 1]
```diff
@@ -9,10 +9,17 @@
 
 payload = {
     "integration_id": integration_id,
-    "run": "tendrl.gluster_integration.flows.CreateVolume",
+    "run": "tendrl.gluster.flows.CreateVolume",
```
Comment (r0h4n) [inline etc/create_sample_job.py]: run: "gluster.flows.CreateVolume"

**Pull request merged**